pmd-go/src/main/java/net/sourceforge/pmd/lang/go/GoLanguageModule.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.go;
+
+import net.sourceforge.pmd.lang.BaseLanguageModule;
+
+/**
+ * Language Module for Go.
+ */
+public class GoLanguageModule extends BaseLanguageModule {
+
+    /** The name. */
+    public static final String NAME = "Golang";
+    /** The terse name. */
+    public static final String TERSE_NAME = "go";
+
+    /**
+     * Create a new instance of Swift Language Module.
+     */
+    public GoLanguageModule() {
+        super(NAME, null, TERSE_NAME, null, "go");
+        addVersion("", null, true);
```

Comment:
the current version of the go Language is `1` according to their docs: https://golang.org/doc/go1compat